pkg/models/model_helpers.go

```diff
@@ -20,6 +22,11 @@ func IntPointer(val int) *int {
 	return &val
 }
 
+// TimePointer returns a pointer to a time.Time input
+func TimePointer(t time.Time) *time.Time {
```

Comment:
this is fine, but could be replaced by `helpers.PointerTo`

Reply:
9a3aca479